mucommander-core/build.gradle

```diff
@@ -37,6 +56,17 @@ dependencies {
 jar {
    from configurations.comprise.collect { it.isDirectory() ? it : zipTree(it).matching {
         include 'org/unix4j/**'
+        include 'com/jediterm/**'
+        include 'com/intellij/**'
+        include 'com/pty4j/**'
+        include 'gnu/trove/**'
+        include 'jtermios/**'
+        include 'purejavacomm/**'
+        include 'kotlin/**'
+        include 'org/jetbrains/**'
+        include 'resources/com/pty4j/**'
+        include 'com/sun/**'
+        include 'org/apache/log4j/**'
```

Comment:
how much data does it add to mucommander-core jar?

Reply:
I will check. The other question is whether `mucommander-core` is a good place to include terminal. Maybe a separate subsystem would be better? Another question is whether it works on Windows/Linux - I'm talking about changes made to osgi `Import-Package`.

Reply:
> how much data does it add to mucommander-core jar?

3MB -> 10MB

Those jediterm related jars take 7.5MB.